### ISSUE TITLE
Text runner is failing to start with illegal option error

### ISSUE THREAD

**Issue by aeneasr** [kevgo/text-runner#1413]
On my macOS machine I am unable to start text-runner because:

```
% npx text-runner
Need to install the following packages:
  text-runner
Ok to proceed? (y) y
readlink: illegal option -- f
usage: readlink [-n] [file ...]
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/foobar/workspace/dist/index.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v17.4.0
```

This appears to be caused by incorrect usage of `readlink` on macOS:

```
% readlink -f
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```

**Comment by kevgo** [comment]
Thanks for the bug report @aeneasr and sorry for the issue! Shame on macOS for having such a different setup of the basic core utilities than pretty much all cloud systems (and Docker) do these days. Apparently macOS started symlinking `readlink` to `stat` recently. I have fixed this problem by getting rid of the dependency on Bash altogether and booting Text-Runner via Node now. I have released Text-Runner v5.0.1 that incorporates the fix. That fixed the issue for me. Please give it a try!

**Comment by aeneasr** [comment]
Thank you a lot :)
